django/views/generic/base.py

```diff
@@ -1,5 +1,6 @@
 import logging
 from functools import update_wrapper
+import inspect
```

Comment:
This should go up before `import logging`.

Reply:
My bad. I didn't sort the imports.